src/envoy/repositories.bzl

```diff
@@ -522,14 +522,17 @@ cc_library(
         "source/**/*.cc",
         "source/**/*.h",
         "include/**/*.h",
-    ], exclude=["source/exe/main.cc"]) + [
+    ]) + [
```

Comment:
removing this exclusion will fail the test.

Add a `cc_library`
```
cc_library(
    name = "envoy-main",
    srcs = [
        "source/exe/main.cc",
    ],
    deps = [
        ":envoy-common",
    ]
)
```

and depends on it from the prototype.